src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/DefaultRazorFormattingServiceTest.cs

```diff
@@ -0,0 +1,383 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting
+{
+    public class DefaultRazorFormattingServiceTest : FormattingTestBase
```

Comment:
You should sprinkle tests that cover the various indentation options as well as disabling formatting entirely

Reply:
Love these tests; however, imagine changing how we format `@functions` directives, aka imagine we start enforcing where the `{` goes (doubt we ever would but it's just an example)? Since the expected output of these tests is in the source updating that output will be a huge manual pain. Something to keep in mind for future iterations (don't need to change anything now).

Reply:
True. In my mind, this is definitely going to change once I figure out how to test HTML formatting because there are no APIs I can call unlike C#.

Reply:
> You should sprinkle tests that cover the various indentation options 

Added. I've skipped one of them due to https://github.com/dotnet/aspnetcore/issues/18996

> as well as disabling formatting entirely

Already added tests in RazorFormattingEndpointTest.cs